docs/app/@theme/theme.module.ts

```diff
@@ -20,6 +20,8 @@ import {
   NbButtonModule,
   NbSelectModule,
   NbInputModule,
+  NbOptionListModule,
```

Comment:
You don't need import modules here unless you need to use components directly in the docs app (not for examples). Docs loads playground app for example where this modules already imported

Reply:
It's used in docs/app/@theme/components/header.component.ts.